modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -87,7 +87,7 @@ storage:
     claim: <1>
 ----
 +
-<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` PVC.
+<1> Leave the `claim field` blank to allow the automatic creation of an `image-registry-storage` Persistent Volume Claim (PVC). The PVC will be generated based on the default storage class. However, be aware that the default storage class might provide ReadWriteOnce (RWO) volumes, such as a RADOS Block Device (RBD), which can cause issues when replicating to more than one replica.
```

Comment:
swap/ will be / is

We try to stay away from future tense/will. 

https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense

Reply:
Sure, will do that!

Reply:
Done